### ISSUE TITLE
Benefit Summary & Service Verification letter - Accessibility issues

### ISSUE THREAD

**Issue by esther-k** [department-of-veterans-affairs/va.gov-team#16510]
## What Happened?
**VoiceOver / TalkBak**

1. With the appropriate screen reader enabled on iOS and Android, the military service info items are being picked up as menu items instead of ReadOnly
     - Note: this is causing the menu items to picked up as selectable by voice control/access and by the Keyboard
2. With the appropriate screen reader enabled on iOS and Android, the a11yHint for the back button is not being picked up 
     - NOTE: Should be updated with [a11y update ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/17185)
3. With TalkBack enabled on Android, the military service info dates are being incorrectly picked up
     - Hearing, for example, "January 01 minus 1,990"
4. With the appropriate screen reader enabled on iOS and Android, the position (place on the list) of items is not being announced
5. With VoiceOver enabled on iOS, the menu item a11yHints are not being picked up
     - Note: This seems to be affected by the [toggle switch bug on iOS](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/16490)
6. With VoiceOver enabled on iOS, the toggle switch states are not being picked up 
     - Note: This seems to be affected by the [toggle switch bug on iOS](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/16490)

**Voice Control / Access** 

7. With Voice Control enabled on iOS, the toggle-able menu items can not be selected
     - Note: This seems to be affected by the [toggle switch bug on iOS](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/16490)

**Keyboard**

8. With Keyboard linked to iOS, the toggle-able menu items can not be selected by either the space bar or enter key 
     - Note: This seems to be affected by the [toggle switch bug on iOS](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/16490)

**Other**

9. Toggle switch bug on iOS is covered in a [separate ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/16490)
     - Note: This bug seems to affect bugs # 5, 6, and 7 on this ticket

## Specs:
- Device: Android - LG LM-Q720 Stylo 5 with TalkBack enabled
- Device: iOS - iPhone XR with VoiceOver enabled
- Bluetooth Keyboard
- User: marcy.nadeau@va.gov

## Steps to Reproduce
**VoiceOver / TalkBak**

For 1: 
1. With the appropriate screen reader enabled on iOS or Android, navigate to the benefits summary & service verification letter page
2. Select one of the items in the military service info section 
3. Note that the screen reader picks these up as menu items

For 3:
1. With TalkBack enabled on Android, navigate to the benefits summary & service verification letter page
2. Select one of the items in the military service info section 
3. Note the dates are picked up as "January 01 minus 1,990" (example)

For 4:
1. With the appropriate screen reader enabled on iOS or Android, navigate to the benefits summary & service verification letter page
2. Select any item that is in a list 
3. Not that it's position on the list is not announced 

## Desired Behavior
**VoiceOver / TalkBak**

For 1:
- With the appropriate screen reader enabled on iOS or Android, the military service info items will be ReadOnly 

For 3:
- With TalkBack enabled on Android, the military service info dates will be read appropriately as "January 01, 1990" (example)

For 4:
- With the appropriate screen reader enabled on iOS or Android, the position of items on a list will be announced 

## Acceptance Criteria
- See desired behavior

## Bug Severity
- 3 - Medium

## Linked Story (s)
- Story #13947

**Comment by digitalken** [comment]
PASS 2/4/2021 - One minor issue persists.  New bug #19519.  Passed everything else.
